components/GameForm/GameForm.js

```diff
@@ -254,6 +271,23 @@ const InputContainer = styled.input`
   }
 `;
 
+const InputImageContainer = styled.input`
+  border: none;
+  margin: 1em 0;
+  height: 3em;
+  padding-left: 1em;
+  font-size: 1em;
+  color: var(--background-primary);
+  background: var(--background-forminput);
+
+  &.file-upload-button {
+    border: 1px solid #ccc;
+    display: inline-block;
+    padding: 6px 12px;
+    cursor: pointer;
+  }
```

Comment:
Does this have any effect? Because I can't find the corresponding part in the component.